src/prettytable/prettytable.py

```diff
@@ -996,6 +999,21 @@ def _set_columns_style(self):
         self.left_padding_width = 0
         self.right_padding_width = 8
 
+    def _set_drawing_style(self):
+
+        self.drawing = True
```

Comment:
Can this be done without introducing a new variable just for this style?

https://github.com/jazzband/prettytable/pull/104 will introduce a bunch of new junction chars, will one of them help here?

Reply:
yes sure, i can use left_junction_char and right_junction_char but how can i include this usage if #104 isn't merged yet?

Reply:
Good questions! We can either wait until #104 is merged, or hoist bits of that code into here.

I think it's better to wait, hopefully #104 will be merged very soon, it's almost ready.

There will probably be merge conflicts whichever is merged first. That one's a bigger and older PR so makes sense to get that in first, will be easier to sort out the conflicts in this smaller one.

Don't worry, I can help with that if needed.

Thank you!

Reply:
I've resolved the conflicts in this PR after the merge of #104.

Do you think you can update this now to replace `self.drawing`?

Reply:
Yes i would like to, but i am not sure if i should bring my fork to the newest version of the master first? or can i edit it directly here? i want to avoid merge conflicts

Reply:
Yes, you should update your fork's branch to match this. Editing directly here is possible too and may be easier, click the pencil edit icon at the top right here:

https://github.com/unconsciou5/prettytable/blob/master/src/prettytable/__init__.py

Reply:
okay, maybe i just refork it and open a new pull request, so i can run the tests too with the new functions

Reply:
wait, i just saw the style is already implemented as "SINGLE_BORDER" this makes the DRAWING one obsolete.

Reply:
They're slightly different, but if `SINGLE_BORDER` suits your purposes then that's good to hear!

`SINGLE_BORDER`

```
┌─────────┬─────────┬─────────┐
│ Field 1 │ Field 2 │ Field 3 │
├─────────┼─────────┼─────────┤
│ value 1 │  value2 │  value3 │
│ value 4 │  value5 │  value6 │
│ value 7 │  value8 │  value9 │
└─────────┴─────────┴─────────┘
```

`DRAWING`

```
│ Field 1 │ Field 2 │ Field 3 │
├─────────┼─────────┼─────────┤
│ value 1 │  value2 │  value3 │
│ value 4 │  value5 │  value6 │
│ value 7 │  value8 │  value9 │
```

Anyway, thank you for your contribution!